uefi-sct/Doc/UserGuide/SecureBootConfigurationGuide.md

```diff
@@ -0,0 +1,105 @@
+
+# Enrolling and Disabling Secure Boot Keys on EDK2-Based UEFI Firmware
+
+This guide outlines the steps for enrolling and disabling Secure Boot keys using UEFI firmware based on EDK2.
+
+## Requirements
+
+- A UEFI firmware implementation based on EDK2
+- Secure Boot keys including:
+  - Platform Key (PK)
+  - Key Exchange Key (KEK)
+  - Signature Database (db)
+  - Forbidden Signature Database (dbx)
+- For instructions on generating and setting up these keys, refer to:
+  - `uefi-sct/Doc/UserGuide/SecureBootTestKeyGenerationAndSetupGuide.md`
+
+## Enrolling Secure Boot Keys
+
+1. **Access the UEFI Setup Utility**
+   - During system boot, press the designated key (e.g., `Esc`, `Del`, `F2`) to enter the firmware setup.
+
+2. **Open the Security or Boot Configuration Menu**
+   - Use arrow keys to navigate.
+   - Look for sections such as `Boot Manager`, `Device Manager`, `Security`, or `Advanced`.
+
+3. **Locate Secure Boot Settings**
+   - Find the `Secure Boot Configuration` section.
+   - This may be under `Device Manager` or a specific `Secure Boot` menu.
+
+4. **Proceed with Key Enrollment**
+   - Follow platform-specific procedures to load PK, KEK, db, and dbx.
+   - Example: For Raspberry Pi, refer to the section “Enrolling Secure Boot Keys on Raspberry Pi Using UEFI Firmware.”
```

Comment:
The Raspberry Pi is typically not booted via UEFI. Why don't we use EDK II as the reference in the documenation?